sources/packages/web/src/components/layouts/Institution/sidebar/HomeSideBar.vue

```diff
@@ -38,6 +40,18 @@ export default defineComponent({
         },
       },
     ]);
+
+    if (institutionState.value.institutionType === "BC Public") {
```

Comment:
We already have `isBCPrivate`  on `InstitutionStateForStore` and I believe that we should add an `isBCPublic` for checks like this in the same way.